notebooks/reinforcement_learning/solutions/contextual_bandits_with_tf_agents.ipynb

```diff
@@ -0,0 +1,1813 @@
+{
```

Comment:
<p>nit: spelling error instanciated -&gt; instantiated here and everywhere else in notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Done!</p>